prov/verbs/src/verbs_domain.c

```diff
@@ -122,16 +122,15 @@ fi_ibv_mr_reg(struct fid *fid, const void *buf, size_t len,
 static int fi_ibv_domain_close(fid_t fid)
 {
 	struct fi_ibv_domain *domain;
-	int ret = FI_SUCCESS;
+	int ret;
 
 	domain = container_of(fid, struct fi_ibv_domain, domain_fid.fid);
 
 	if (domain->rdm) {
-		errno = 0;
-		rdma_destroy_ep(domain->rdm_cm->listener);
-		if (errno) {
+		ret = rdma_destroy_ep(domain->rdm_cm->listener);
+		if (ret) {
```

Comment:
Yes. It's interesting that the man pages available on the internet (linux.die.net for example) say rdma_destroy_ep returns int when it doesn't.